static/app/views/starfish/queries/useSpanMetrics.tsx

```diff
@@ -51,17 +47,15 @@ export const useSpanMetrics = <Fields extends MetricsProperty[]>(
 };
 
 function getEventView(
-  filters: SpanMetricsQueryFilters = {},
+  search: MutableSearch | undefined,
   fields: string[] = [],
   sorts: Sort[] = [],
   pageFilters: PageFilters
 ) {
-  const query = MutableSearch.fromQueryObject(filters);
-
   const eventView = EventView.fromNewQueryWithPageFilters(
     {
       name: '',
-      query: query.formatString(),
+      query: search?.formatString() ?? '',
```

Comment:
Will there be issues with this being an empty string?

Reply:
Good question! I just checked and a missing `query=`, `query=` and `query=""` all have the same result 👍🏻